stretch/Dockerfile

```diff
@@ -35,11 +35,11 @@ ENTRYPOINT ${STEAMCMDDIR}/steamcmd.sh \
 			Port=$PORT QueryPort=$QUERYPORT RCONPORT=$RCONPORT FIXEDMAXPLAYERS=$FIXEDMAXPLAYERS RANDOM=$RANDOM
 
 # Expose ports
-EXPOSE 7787\udp
-EXPOSE 7788\udp
-EXPOSE 27165\tcp
-EXPOSE 27165\udp
-EXPOSE 27166\tcp
-EXPOSE 27166\udp
-EXPOSE 21114\tcp
-EXPOSE 21114\udp
+EXPOSE 7787/udp
+EXPOSE 7788/udp
+EXPOSE 27165/tcp
+EXPOSE 27165/udp
+EXPOSE 27166/tcp
+EXPOSE 27166/udp
```

Comment:
Why do you EXPOSE port 7788 and 27166?

Also as stated before please collapse into one EXPOSE line and use \ to add linebreaks if necessary.

Reply:
I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open
I also exposed every port by one line for a better overview.

Reply:
> 
> 
> I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open

Yes, and QueryPort+1 is for additional game server instances. If you run `lsof` in the container you will never see those ports binded unless specified. This is the same for all Unreal Engine games, they don't have a fallback port mechanic. The Wiki documentation is just off by a bit.

> I also exposed every port by one line for a better overview.

Yes, which is a good thing but there is no need to use a new EXPOSE every new line. This is way better:
```
EXPOSE 7787/udp \
	27165/tcp \
	27165/udp \
	21114/tcp \
	21114/udp
```

Reply:
Ok changed it